### PR TITLE
🙊🙊 Add `docker` as a supported SDK for Spaces

### PR DESCRIPTION
**Pull request by julien-c**

### CODE REVIEW

**Comment by Wauplin** [comment]
🔥🔥🔥

**Comment by camenduru** [comment]
😲😲😲

**Review by osanseviero — APPROVED** [review]
Nice! :fire:

File: src/huggingface_hub/hf_api.py [hunk 1]
```diff
@@ -1801,7 +1801,7 @@ def create_repo(
             exist_ok (`bool`, *optional*, defaults to `False`):
                 If `True`, do not raise an error if repo already exists.
             space_sdk (`str`, *optional*):
-                Choice of SDK to use if repo_type is "space". Can be "streamlit", "gradio", or "static".
+                Choice of SDK to use if repo_type is "space". Can be "streamlit", "gradio", "docker", or "static".
```
Comment (Wauplin) [inline src/huggingface_hub/hf_api.py:1804]:
```suggestion
                Choice of SDK to use if repo_type is "space". Can be "gradio", "streamlit", "docker", or "static".
```
just to keep consistency with `SPACES_SDK_TYPES` definition

**Comment by julien-c** [comment]
arg i didn't check the commit title before merging :(

**Review by Wauplin — APPROVED** [review]
Since is such a small update compared to the feature it enables :D 

Look good to me. Tested it locally and works fine.

File: src/huggingface_hub/hf_api.py (hunk 1 above)
Comment (Wauplin) [reply src/huggingface_hub/hf_api.py:1804]: oops, I'm too late. Nevermind :)

Comment (julien-c) [reply src/huggingface_hub/hf_api.py:1804]: oops sorry!

**Pull request merged**